test/gtest/uct/uct_test.h

```diff
@@ -153,6 +153,7 @@ std::ostream& operator<<(std::ostream& os, const resource* resource);
     UCT_TEST_IB_TLS, \
     ugni, \
     mm, \
```

Comment:
I think I updated all the relevant places. At least I don't see any other places where cma is missing.